components/cart/cart.css

```diff
@@ -0,0 +1,79 @@
+/* cart */
+.display-cart-section{
+    display: flex;
+    flex-direction: row;
+    max-width: 100%;
+    margin: 3rem 2rem; 
+    padding: 3rem;
+}
+.grid-3-column-layout-cart{
```

Comment:
it seems like here you are trying to make a 3 column layout using grid but I can see only 1 column of width 3fr made, to make a 3 column you could use g-tem-colms: 1fr 1fr 1fr , defining width for each column or g-temlate-colms: repeat(3, 1fr)

Reply:
Or if it is just a single colum layout you could change the classnames

Reply:
What you have done here is made 1 colum with width 3fr and 1 row with width 1fr

Reply:
ok I will look into it.